PersonApi/PersonApi.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Hackney.Core.Middleware" Version="1.30.0" />
     <PackageReference Include="Hackney.Core.Sns" Version="1.30.0" />
     <PackageReference Include="Hackney.Core.Validation" Version="1.30.0" />
+    <PackageReference Include="Hackney.Shared.Person" Version="0.2.2-preview0004" />
```

Comment:
Seeing as we need to update the version, we shouldn't merge this PR until the version is updated